vendor/k8s.io/csi-translation-lib/translate.go

```diff
@@ -125,16 +136,20 @@ func GetInTreePluginNameFromSpec(pv *v1.PersistentVolume, vol *v1.Volume) (strin
 		}
 		return "", fmt.Errorf("could not find in-tree plugin name from persistent volume %v", pv)
 	} else if vol != nil {
-		// TODO(dyzz): Implement inline volume migration support
-		return "", errors.New("inline volume migration not yet supported")
+		for _, curPlugin := range inTreePlugins {
```

Comment:
these are all vendored in changes (bringing external-provisioner up to HEAD of the translation lib). Arguably this could be a seperate PR since it does actually introduce functional changes